src/Nethermind/Nethermind.Synchronization/ParallelSync/MultiSyncModeSelector.cs

```diff
@@ -555,8 +562,8 @@ private bool ShouldBeInStateSyncMode(Snapshot best)
 
         private bool CanBeInSnapRangesPhase(Snapshot best)
         {
-            long peerBlock = best.Peer.Block;
-            bool isCloseToHead = peerBlock >= best.Header && (peerBlock - best.Header) < Constants.MaxDistanceFromHead;
+            bool isCloseToHead = best.TargetBlock >= best.Header &&
```

Comment:
so before we were closeToTheHead when we were close to the Peer. However, now we will relay on lastNewPayload, so we can start snap sync before we reach the head of the chain. Requires lots of testing :)

Reply:
Ah good catch, we potentially might have issues when our CL is behind?